commit f81040276a65780fd40cd1a964f1d2c1e7959f75
Author: Chen Hanxiao <chenhx.fnst@fujitsu.com>
Date:   Thu Feb 8 16:06:26 2024 +0800

    nfsd: clean up comments over nfs4_client definition
    
    nfsd fault injection has been deprecated since
    commit 9d60d93198c6 ("Deprecate nfsd fault injection")
    and removed by
    commit e56dc9e2949e ("nfsd: remove fault injection code")
    
    So remove the outdated parts about fault injection.
    
    Signed-off-by: Chen Hanxiao <chenhx.fnst@fujitsu.com>
    Reviewed-by: Jeff Layton <jlayton@kernel.org>
    Signed-off-by: Chuck Lever <chuck.lever@oracle.com>

diff --git a/fs/nfsd/state.h b/fs/nfsd/state.h
index be02bf1a16bd..0c8ec578ba7e 100644
--- a/fs/nfsd/state.h
+++ b/fs/nfsd/state.h
@@ -334,8 +334,9 @@ enum {
  * 0. If they are not renewed within a lease period, they become eligible for
  * destruction by the laundromat.
  *
- * These objects can also be destroyed prematurely by the fault injection code,
- * or if the client sends certain forms of SETCLIENTID or EXCHANGE_ID updates.
+ * These objects can also be destroyed if the client sends certain forms of
+ * SETCLIENTID or EXCHANGE_ID operations.
+ *
  * Care is taken *not* to do this however when the objects have an elevated
  * refcount.
  *
@@ -343,7 +344,7 @@ enum {
  *
  * o Each nfs4_clients is also hashed by name (the opaque quantity initially
  *   sent by the client to identify itself).
- * 	  
+ *
  * o cl_perclient list is used to ensure no dangling stateowner references
  *   when we expire the nfs4_client
  */